package.json

```diff
@@ -64,13 +51,17 @@
     "react-final-form-listeners": "^1.0.2",
     "react-router-prop-types": "^1.0.4",
     "redux-form": "^8.3.7",
-    "zustand": "^3.7.2"
+    "zustand": "^4.1.1"
   },
   "peerDependencies": {
-    "react": "^17.0.2",
-    "react-intl": "^5.8.0",
+    "@folio/stripes": "^9.1.4",
+    "react": "^18.2.0",
+    "react-intl": "^6.4.4",
     "react-query": "^3.6.0",
     "react-router": "^5.0.1",
     "react-router-dom": "^5.0.1"
+  },
+  "resolutions": {
+    "moment": "~2.29.4"
```

Comment:
I think we are moving towards using dayjs instead of moment js and already began using dayjs for any new development. Please recheck on this aspect from stripes channel.

Reply:
Thanks for this, I will create a separate ticket for this, since this ticket is not about the transition from `moment` to `dayjs`.

Reply:
Create a task for that https://folio-org.atlassian.net/browse/UISINVCOMP-2